### DISCUSSION TITLE
View entries in entry index even if not editable per other-author permission

### DISCUSSION THREAD

**Discussion by brandonkelly** [craftcms/cms#8881] [Ideas]
### Description

If an author doesn’t have permission to edit other authors’ entries, they should at least be able to view those entries in the Entries index. That way they will still be able to see the hierarchy for Structure sections, which is pretty awkward right now as pointed out by https://github.com/craftcms/cms/issues/6516#issuecomment-667918547:

> when you do choose a parent not authored by the current user, it shows it on it's own in the list with no indication as to it's parent(s)

**Comment by JoshCoady** [comment]
I dont know if i should post this as a separate issue, but I would also like to be able to allow users to view any entries they are not allowed to edit. I imagine this as having `View entries` as the top-level permission checkbox under each section and then `Edit entries` is under that.

Our current use-case is we have a multi-site that is for a site with multiple subdomains. Certain authors are only allowed to edit content on certain subdomains (sites), but we'd like them to be able to see all of the content because they might want to locate similar content on other subdomains to link to it and other things of that nature.

But I guess this is a little more complicated because we'd also need the ability to set groups/permissions per site (like put someone in the Viewer group for sites A and B and in the Writer group for sites C and D).

**Comment by brandonkelly** [comment]
@JoshCoady Sounds slightly different if you need the permissions to be site-based. Can you post as a new issue?
